src/firefly/js/templates/lightcurve/wise/WisePlotRequests.js

```diff
@@ -26,15 +27,15 @@ export function makeWisePlotRequest(table, rowIdx, cutoutSize) {
     /*the following should be from reading in the url column returned from LC search
      we are constructing the url for wise as the LC table does
      not have the url column yet
-     It is only for WISE, using default cutout size 0.3 deg
+     set default cutout size 5 arcmin
      const url = `http://irsa.ipac.caltech.edu/ibe/data/wise/merge/merge_p1bm_frm/${scangrp}/${scan_id}/${frame_num}/${scan_id}${frame_num}-w1-int-1b.fits`;
      */
     const serverinfo = 'http://irsa.ipac.caltech.edu/ibe/data/wise/merge/merge_p1bm_frm/';
     const centerandsize = cutoutSize ? `?center=${ra},${dec}&size=${cutoutSize}&gzip=false` : '';
     const url = `${serverinfo}${scangrp}/${scan_id}/${frame_num}/${scan_id}${frame_num}-w${band}-int-1b.fits${centerandsize}`;
     const plot_desc = `WISE-${frameId}`;
     const reqParams = WebPlotRequest.makeURLPlotRequest(url, plot_desc);
-    const title = 'WISE-' + frameId + (cutoutSize ? ` size: ${cutoutSize}(deg)` : '');
+    const title = 'WISE-' + frameId + (cutoutSize ? ` size: ${cutoutSize}(arcmin)` : '');
```

Comment:
This was the older version of image retrieve via url, it is not used for WISE anymore. Since we kept code here, I will convert the size to degree just to be consistent with the IBE image retrieve.